### ISSUE TITLE
Checkout PRO: Unable to review data when marking purchase as suspected

### ISSUE THREAD

**Issue by lucasexe** [mercadopago/sdk-js#36]
**Describe the bug**
Hi,
In the last few days, I faced an error using a credit card. MercadoPago considered the purchase as high risk and showed a message like:
"The order may be refused if you don't review your data.  <button to review data>"

When I clicked in the button all the frame was blocked by the browser, saying that the frame was refused.

**To Reproduce**
I am not able to reproduce the issue right now (the orders are being approved when I use the credit card I have).
But when the issue happened I've copied the button link:

https://auth-identity-webview.mercadopago.com/mlb/account/identity/validation?flow=ss_proof_of_life&skipLanding=true&reference_id=checkout_off_basic_v2&transactionId=S0M7iSaqAAiUwImT8CJdcUpbYKmwagK3#initiative=second-score&from=remedy&pid=17169559509&flow=proof_of_life_last_chance&source=congrats_page&reference_id=checkout_off_basic_v2

I hope this is enough to identify the issue.

**Expected behavior**
Open the review data frame properly.

**Environment (please complete the following information):**
- Browser: "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.128 Safari/537.36"
- Navigator language: pt-BR

Thanks

**Comment by NicholasPedroso** [comment]
Hi, this problem should be solved by now.
Closing the issue.

Please feel free to open a new issue if the problem persists
